docker/base.Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 # Install base dependencies that Python or Go would require
 RUN apt-get -qq update && apt-get -qq install -y \
+    apt-utils \
```

Comment:
Can you explain why apt-utils is now necessary?

Reply:
With the QEMU build building for multiple flavors (adding arm64 in this change), I ran into an issue where apt-utils needed to be re-installed in order to work.